packages/api/src/controllers/contacts.controller.ts

```diff
@@ -21,6 +23,13 @@ export class ContactsController {
         return this.contactsService.findAll(getContactsRequest.id);
     }
 
+    @UseGuards(JwtAuthGuard, CanAccessResourceGuard)
+    @Get("users/:id/contacts/:name")
+    async findSpecific(@Param() params: GetSpecificContactsRequest): Promise<Contact[]> {
```

Comment:
Standard naming convention would just be this

```suggestion
    async find(@Param() params: GetSpecificContactsRequest): Promise<Contact[]> {
```